lib/drivers/medtronic600/NGPHistoryParser.js

```diff
@@ -942,6 +941,23 @@ class NGPHistoryParser {
     }
   }
 
+  findMatchingClosedLoopBGReadings(bgReadingEvent) {
+    // Find all of the ClosedLoopBloodGlucoseReadingEvents with the same BG value that
+    // occurred within 15 minutes of the BG_READING_RECEIVED.
```

Comment:
Yeah, 15 minutes seem reasonable. That's the same amount of time we use to deduplicate manual and linked values on Animas and Medtronic.

Reply:
Yeah. 15 minutes is also in the CLP code.